yellowbrick/features/base.py

```diff
@@ -190,6 +199,7 @@ def __init__(self, ax=None, features=None, classes=None, color=None,
         # Visual Parameters
         self.color = color
         self.colormap = colormap
+        self.target=target
```

Comment:
Likely we'll need some kind of validation that the target string supplied by the user is correct and raise a `YellowbrickValueError` on `__init__` if not. E.g. if the string is misspelled. Above I suggest an enum as one mechanism to do this, though you could use a set and some string manipulation as well if it is preferable.

Reply:
I can add it but it will ultimately be duplicate for the exception in line 278.

Reply:
Yeh, that's true - but it will be a little backup in case something goes really weird.